CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 -   #1720 : Add support for `Ellipsis` as the only index for an array.
--   #1787 : Ensure STC v5.0 is installed with Pyccel.
+-   #1787 : Ensure STC v5.0 (<ef322aeb21c682a02253ce6df524307d142294c7>) is installed with Pyccel.
```

Comment:
`<EXPLANATION>` is often used as a placeholder. I think the short SHA should be sufficient here
```suggestion
-   #1787 : Ensure STC v5.0 (`ef322ae`) is installed with Pyccel.
```